scanner.py

```diff
@@ -2740,6 +2740,9 @@ class ScannerWrapper:
     def __init__(self, scanner: Scanner):
         self.scanner = scanner
 
+    def get_status(self, eventtime=None) -> "dict[str, str]":
```

Comment:
eventtime is of type float
You can see how different status classes are made in typings, I usually call them _Status. So maybe it makes sense to have a _ProbeStatus typeddict returned here?